cmd/cluster/dynatrace/hcpMgCmd.go

```diff
@@ -0,0 +1,300 @@
+package dynatrace
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+func NewCmdHCPMustGather() *cobra.Command {
+	hcpMgCmd := &cobra.Command{
+		Use:               "must-gather <cluster-id>",
+		Aliases:           []string{"mg"},
+		Short:             "Gather all Pod logs and Application event from HCP",
+		Args:              cobra.ExactArgs(1),
+		DisableAutoGenTag: true,
+		Run: func(cmd *cobra.Command, args []string) {
+			err := mustGather(args[0])
+			if err != nil {
+				cmdutil.CheckErr(err)
+			}
+		},
+	}
+
+	hcpMgCmd.Flags().IntVar(&since, "since", 10, "Number of hours (integer) since which to pull logs and events")
+	hcpMgCmd.Flags().IntVar(&tail, "tail", 100, "Last 'n' logs and events to fetch ")
```

Comment:
**issue:** I don't think we need the tail flag if we have a `since` flag. I would expect all logs in the time interval to be gathered when there's a `since` flag. This would cause confusion in my opinion.

**issue:** 100 lines seems way too low as default. A standard must-gather will get all the available logs, can we do this here as well?